milli/src/update/index_documents/mod.rs

```diff
@@ -345,9 +345,9 @@ where
         self.index.put_documents_ids(self.wtxn, &all_documents_ids)?;
 
         self.execute_prefix_databases(
-            word_docids,
-            word_pair_proximity_docids,
-            word_position_docids,
+            word_docids.unwrap(),
+            word_pair_proximity_docids.unwrap(),
+            word_position_docids.unwrap(),
```

Comment:
We may instead match this to compute or not each concerned prefix database, no?
something like:
```rust
if let Some(word_pair_proximity_docids) = word_pair_proximity_docids {
   // compute word prefix pair proximity database
}

Reply:
I prefer your solution indeed but I think we always generate the grenad files, they are just empty, am I right?